lib/src/hook.dart

```diff
@@ -41,11 +49,18 @@ class _UseConnectivityState extends HookState<void, _UseConnectivity> {
     if (hook.config != null) {
       Connectivity.instance.setCustomConfig(hook.config!);
     }
+
+    // Start monitoring connectivity.
     Connectivity.instance.start(
       context: hook.context,
       fallBackScreen: hook.fallBackScreen,
       connectivityDisplayType: hook.displayType,
     );
+
+    // Set custom configuration if provided.
```

Comment:
this code has been changed so the comment needs to moved to the if(hook.config != null) above this
You might need to rebase with master because the change might not be on your branch yet